specification/securityinsight/preview/2019-01-01/SecurityInsight.json

```diff
@@ -0,0 +1,366 @@
+      "properties": {
+        "code": {
+          "readOnly": true,
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        }
+      }
+    },
+    "AlertRule": {
+      "type": "object",
+      "description": "Alert rule.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+			"x-ms-client-flatten": true,
+			"description": "Alert rule properties",
+			"$ref": "#/definitions/AlertRuleProperties"
+        }
+      }
+    },
+    "AlertRuleProperties": {
+      "type": "object",
+      "description": "Alert rule property bag.",
+      "properties": {
+        "query": {
+          "type": "string",
+          "description": "The query that will create alerts for this rule."
+        },
+        "period": {
+          "type": "string",
+          "description": "The period that the alert will look at."
+        }
+      }
+    },
+    "Resource": {
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionId": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$",
+      "type": "string",
+      "description": "Azure subscription ID"
+    },
+    "ResourceGroupName": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the resource group within the user's subscription. The name is case insensitive.",
+      "pattern": "^[-\\w\\._\\(\\)]+$",
+      "minLength": 1,
+      "maxLength": 90,
+      "x-ms-parameter-location": "method"
+    },
+    "ApiVersion": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "2019-01-01-preview"
+      ],
+      "description": "API version for the operation"
+    },
+	"OperationalInsightsResourceProvider": {
+		"name": "operationalInsightsResourceProvider",
+		"in": "path",
+		"required": true,
+		"type": "string",
+		"description": "The namespace of workspaces resource provider- Microsoft.Operationalinsights.",
```

Comment:
Think the "I" in "Insights" probably should be capitalized too

Reply:
You right, updated